data/Nishitbaria.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Nishit Baria",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**Open Source** Advocate | Web-Dev | Space Lover ",
+  "tags": ["Open Source", "c++", "Java", "SQL", "Open Source "],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/Nishitbaria1" },
+    { "icon": "FaGithub", "url": "https://github.com/Nishitbaria" },
+    { "icon": "FaLink", "url": "https://nishithacks.netlify.app/" }
+  ],
+  "avatar": "https://github.com/Nishitbaria.png",
+  "links": [
+    {
+      "name": "Learing in Public and tweet about it ",
+      "url": "https://twitter.com/Nishitbaria1",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow my Github to see my contribution",
+      "url": "https://github.com/Nishitbaria",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/nishitbaria/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Personal website",
+      "url": "https://nishithacks.netlify.app/",
+      "icon": "FaGlobe"
+    }, 
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/@Nishitbaria",
+      "icon": "FaYouTube"
```

Comment:
Everything looks great just this youtube icon needs to change to `FaYoutube`

Reply:
Hey Aadarsh, I solve this problem in short time